alot/commands/globals.py

```diff
@@ -966,26 +967,22 @@ def _set_base_attributes(self):
         # set forced headers for separate parameters
         if self.sender:
             self.envelope.add('From', self.sender)
-        if self.subject:
```

Comment:
to and subject are set in __apply anyway so we don't need these.

Reply:
see above. are they added to non-empty values?